### DISCUSSION TITLE
코멘트상태 없애기

### DISCUSSION THREAD

**Discussion by dongho108** [woowacourse-teams/2022-ternoko#275] [Show and tell]
현재 면담 사전메일이 발송되면서 인터뷰 상태가 FIX로 변하고,
interview endTime이 되면 comment 상태로 바꾸려고 했습니다.

그런데, endTime시에 어떻게 comment 상태로 바꿔줄 수 있을까요..?
언제 어떻게 cron을 동작시켜야할지 감도 안옵니다.

그래서 comment 상태를 없애고 사전메일을 보내면서 FIX가 되면 그때부터 어차피 인터뷰는 수정이 안되니까 코멘트를 달 수 있는 상태로 가면 어떨까요..?

1. 30분마다 endTime 면담을 조회해 상태를 바꾼다.
2. comment 상태 없앤다.
    1. 프론트에서 버튼만 안보여준다. endTime에서만 보여준다.

**Comment by dongho108** [comment]
일단 COMMENT 없애고 FIX상태일때 코멘트를 달 수 있도록 정해졌습니다!

**Comment by her0807** [comment]
예를 들면 배달을 시켰는데, 배달이 오지 않은 상태에서, 음식의 맛을 모른채 리뷰를 남길 수 있게 되는 것과 같네요 ..?
